components/yaml/introduction.rst

```diff
@@ -250,6 +233,74 @@ representation of the object.
     parsers will likely not recognize the ``php/object`` tag and non-PHP
     implementations certainly won't - use with discretion!
 
+Handling Invalid Types
+......................
+
+By default the parser will encode invalid types as "null". You can make the
+parser throw exceptions by using the ``PARSE_EXCEPTION_ON_INVALID_TYPE`` flag:
+
+    $yaml = '!php/object:O:8:"stdClass":1:{s:5:"foo";s:7:"bar";}';
+    Yaml::parse($yaml, Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE); // throws an exception
+
+Inversely you can use ``DUMP_EXCEPTION_ON_INVALID_TYPE`` when dumping:
+
+    $data = new \stdClass(); // by default objects are invalid.
+    Yaml::parse($data, Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE); // throws an exception
+
+Using Objects for Maps
+......................
+
+.. versionadded:: 2.7
+    Support for parsing mappings as objects was introduced in Symfony 2.7.
+
+Yaml :ref:`mappings <yaml-format-collections>` are basically associative
+arrays. You can instruct the parser to return mappings as objects (i.e.
+``\stdClass`` instances) by using the ``PARSE_OBJECT_FOR_MAP`` flag.
+
+    $object = Yaml::parse('{"foo": "bar"}', Yaml::PARSE_OBJECT_FOR_MAP);
+    echo get_class($object); // stdClass
+    echo $object->foo; // bar
+
+and inversely, to encode objects as maps use ``DUMP_OBJECT_AS_MAP``:
+
+    $yaml = Yaml::dump($object, 2, 4, Yaml::DUMP_OBJECT_AS_MAP);
+    echo $yaml; // { foo: bar }
+
+Parsing to DateTime
+...................
+
+By default the YAML parser will encode unquoted strings that look like dates
```

Comment:
`unquoted strings that look like dates` If possible, can we please add more precision to this statement? Which are the formats that activate the date transform?